src/script/engine.zig

```diff
@@ -358,6 +363,82 @@ pub const Engine = struct {
         try self.stack.pushByteArray(value);
     }
 
+    /// OP_NIP: Removes the second-to-top stack item
+    ///
+    /// # Returns
+    /// /// - "EngineError.StackUnderflow": if initial stack length < 2
```

Comment:
Incomplete. I can also error if the allocator runs out of memory.
So don't present it as the `return` section, just write "will return an error if initial stack length < 2"